components/list/list-item-drag-drop-mixin.js

```diff
@@ -559,14 +601,34 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 
 		const rootList = this._getRootList(this);
 		const selectionInfo = rootList.getSelectionInfo(rootList.dragMultiple);
-
 		if (rootList.dragMultiple && selectionInfo.keys.length > 1) {
+			const lazyLoadListItems = this._getFlattenedListItems().lazyLoadListItems;
 			let dragImage = this.shadowRoot.querySelector('d2l-list-item-drag-image');
 			if (!dragImage) {
 				dragImage = document.createElement('d2l-list-item-drag-image');
 				this.shadowRoot.appendChild(dragImage);
 			}
+			let includePlus = false;
+			if (lazyLoadListItems.size > 0) {
+				for (const selectedItemKey of selectionInfo.keys) {
+					if (lazyLoadListItems.has(selectedItemKey)) {
+						includePlus = true;
+						break;
+					}
+				}
+			}
 			dragImage.count = selectionInfo.keys.length;
+			dragImage.includePlusSign = includePlus;
+			e.dataTransfer.setDragImage(dragImage, 24, 26);
+		} else if (rootList.dragMultiple && this.expandable) {
+			const flattenedListItems = this._getFlattenedListItems(this);
+			let dragImage = this.shadowRoot.querySelector('d2l-list-item-drag-image');
```

Comment:
Looks like we could factor this out into a function to avoid the duplication from above. It could be done inline in this method or at the class level.
```javascript
const getDragImage = (count, includePlusSign) => {
	let dragImage = this.shadowRoot.querySelector('d2l-list-item-drag-image');
	if (!dragImage) {
		dragImage = document.createElement('d2l-list-item-drag-image');
		this.shadowRoot.appendChild(dragImage);
	}
	dragImage.count = count;
	dragImage.includePlusSign = includePlusSign;
	return dragImage;
};
```